test/language/statements/for-in/identifier-let-allowed-as-lefthandside-expression-not-strict.js

```diff
@@ -1,11 +1,12 @@
 // Copyright (C) 2011 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 13.6.0.1
 description: >
-    for declaration:
     identifier "let" allowed as lefthandside expression
+esid: sec-iteration-statements
```

Comment:
This is awesome, it's valuable to update these, because it gives anyone reading this test a way to cross reference the test contents with the specification

Reply:
¯\_(ツ)_/¯